### ISSUE TITLE
Wrong hashes for special charaters

### ISSUE THREAD

**Issue by hgschwibbe** [gdi2290/angular-md5#5]
The md5.createHash function generates wrong hashes for special charaters like üäöÜÄÖ.

**Comment by pawellen** [comment]
same in polish characters:(

**Comment by qinjx** [comment]
yes, because there is no UTF8 supporting, maybe Google's crypto-js can help you: https://code.google.com/p/crypto-js/
